docs/docs/benchmark.md

```diff
@@ -1,96 +1,164 @@
+    print('URL for run: %s/run/%d' %(openml.config.server,run.run_id))
+    ```
+
+!!! Java (with WEKA)
+  ``` java
+  public static void runTasksAndUpload() throws Exception {
+    OpenmlConnector openml = new OpenmlConnector();
+    // obtain the benchmark suite
+    Study benchmarksuite = openml.studyGet("OpenML-CC18", "tasks");
+    Classifier tree = new REPTree(); // build a Weka classifier
+    for (Integer taskId : benchmarksuite.getTasks()) { // iterate over all tasks
+      Task t = openml.taskGet(taskId); // download the OpenML task
+      Instances d = InstancesHelper.getDatasetFromTask(openml, t); // obtain the dataset
+      openml.setApiKey("FILL_IN_OPENML_API_KEY");
+      int runId = RunOpenmlJob.executeTask(openml, new WekaConfig(), taskId, tree);
+      Run run = openml.runGet(runId);   // retrieve the uploaded run
+    }
   }
-}
-```
+  ```
 
-### R (with mlr)
-``` r
-library(OpenML)
-lrn = makeLearner('classif.rpart') # construct a simple CART classifier
-task.ids = getOMLStudy('OpenML100')$tasks$task.id # obtain the list of suggested tasks
-for (task.id in task.ids) { # iterate over all tasks
-  task = getOMLTask(task.id) # download single OML task
-  data = as.data.frame(task) # obtain raw data set
-  run = runTaskMlr(task, learner = lrn) # run constructed learner
-  setOMLConfig(apikey = 'FILL_IN_OPENML_API_KEY')
-  upload = uploadOMLRun(run) # upload and tag the run
-}
+!!! R (with mlr)
+  ``` r
+  library(OpenML)
+  lrn = makeLearner('classif.rpart') # construct a simple CART classifier
+  task.ids = getOMLStudy('OpenML-CC18')$tasks$task.id # obtain the list of suggested tasks
+  for (task.id in task.ids) { # iterate over all tasks
+    task = getOMLTask(task.id) # download single OML task
+    data = as.data.frame(task) # obtain raw data set
+    run = runTaskMlr(task, learner = lrn) # run constructed learner
+    setOMLConfig(apikey = 'FILL_IN_OPENML_API_KEY')
+    upload = uploadOMLRun(run) # upload and tag the run
+  }
 ```
 
+### Retrieving all runs on a benchmarking suites:
+- API call /study/name/<studyname> --> list of run ids
+
 ## Creating new benchmark suites
 The set of datasets on OpenML.org can <a href="https://www.openml.org/new/data">easily be extended</a>, and additional OpenML benchmark suites,
 e.g., for regression and time-series data, can easily be created by defining sets of datasets according to specific needs.
 This is done by:
-<ul>
-<li><a href="new/study">Creating a new study</a> with the name of the benchmark as the alias.</li>
-<li>Adding new tasks to the study by <i>tagging</i> them with the name of the benchmark.</li>
-</ul>
+* [https://www.openml.org/new/study](Creating a new study) with the name of the benchmark as the alias.
+* Adding new tasks to the study by _tagging_ them with the name of the benchmark.
+
+Detailed steps:
+- Create a new study with name and description
+- Set flag to "private" - we dont want people adding tasks here
+- Set "suite" flag
```

Comment:
Is this there already?

Reply:
I can't see this in the API docs, so I assume not?